IoT_master/tools/desks_reservation_func.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import print_function
+import pickle
+import os.path
+import yaml
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from os import path
+
+# If modifying these scopes, delete the file token.pickle.
+SCOPES = ['https://www.googleapis.com/auth/calendar']
+
+
+def calendar_config():
+    '''
+    Open a configuration file and read iframe,
+    file with credentials for Calendar API and
+    calendarID
+    :return: List with iframe, api_credentials, calendarID strings
+    :rtype: List
+    '''
+    if(path.exists("../config.yml")):
+        with open('../config.yml', 'r') as file_config:
```

Comment:
Usage of '../config.yml' may be a bit risky if function will be called in other context. If it should call file that's in directory above directory with this file then you can supersede dots with `dir_path = os.path.dirname(os.path.realpath(__file__))` which will give you always directory of your file.